ansible/roles/build-grsec-kernel/tasks/ccache.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Install ccache.
+  apt:
+    name: ccache
+    state: present
+  sudo: yes
+
+- name: Increase ccache cache size.
+  command: ccache -M 5G
```

Comment:
Might be overkill; after a few recompiles on the same box, you should check the cache size and hit rate with `ccache -s` to get a sense of the optimum size.